site/content/docs/5.0/components/stepped-process.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Stepped process
+description: Stepped process bar used for multiple steps forms process
+group: components
+toc: true
+---
+
+## How to use
+
+Use a `nav` element with `.stepped-process` class, containing an ordered list `<ol>`  with `.stepped-process-item` class on list items.
+
+Be careful to choose short labels when using multiple steps process, otherwise label will be cut to preserve inline layout. More information can be carried on the `title` attribute of the `.stepped-process-link`.
+
+Use `.current` class to define current step, alongwith with `aria-current="step"`.
+
+## Example
+
+{{< example >}}
+<nav class="stepped-process" aria-label="Checkout process">
+  <p class="float-left mt-2 mr-2 font-weight-bold d-sm-none">Step</p>
```

Comment:
```suggestion
  <p class="float-end mt-2 me-2 fw-bold d-sm-none">Step</p>
```

Beware that some utilities got renamed in v5: you should have seen the difference visually (e.g. not inline on small screens, and not bold either).

Reply:
Fact is I didn't see a thing, probably because of the `d-sm-none` so I didn't test all breakpoints. My bad...
Secondly I didn't look for changes on utilities, my bad twice.

(In our case, the label should be at the start, I corrected accordingly to this case)

Reply:
Yup, that's what reviews and tests are for ;)